pyproject.toml

```diff
@@ -30,6 +30,7 @@ pytest-freezegun = "0.4.1"
 pytest-mock = "3.2.0"
 requests-mock = "1.8.0"
 black = "19.10b0"
+pre-commit = "2.6.0"
```

Comment:
Just wondering: is poetry responsible for installing pre-commit hooks? If yes, then we might need to update here: https://github.com/scanapi/scanapi/blob/master/.pre-commit-config.yaml#L3
Made a PR on black project about 10 days ago. They no longer have the "stable" tag, which was causing my pre-commit install to fail.

Reply:
> Just wondering: is poetry responsible for installing pre-commit hooks? If yes, then we might need to update here: https://github.com/scanapi/scanapi/blob/master/.pre-commit-config.yaml#L3
> Made a PR on black project about 10 days ago. They no longer have the "stable" tag, which was causing my pre-commit install to fail.

@vinigfer Good catch! Even in their doc the `stable` is still there. Good to know. I updated it. Would you mink to take another look if it is all good?

Reply:
Really? 🤔 
Hum ... the PR I made only changed README.md
Guess will check the docs later and open a new PR

I'll review it again in a couple of minutes ;)